docs/src/componentDocs/DrawerFooter/examples/DrawerFooterExample.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  Drawer,
+  DrawerBody,
+  DrawerFooter,
+  DrawerNavItem,
+} from '@brightlayer-ui/react-native-components';
+import {ExampleShowcase} from '../../../shared';
+import {Text} from 'react-native-paper';
+import {Image, View} from 'react-native';
+import eatonLogo from '../images/eatonLogo.png';
+import eatonCopyrightText from '../images/EatonCopyrightText.png';
+
+export const DrawerFooterExample = (): JSX.Element => (
+  <ExampleShowcase>
+    <Drawer style={{width: 320, margin: 'auto'}}>
+      <DrawerBody>
+        <DrawerNavItem
+          itemID={'item1'}
```

Comment:
use `itemId={'account'}` instead of `item1`. same for other ItemID.

Reply:
Added as per suggestion